### PR TITLE
feat(JGU-91): added User Preferences documentation

### PR DESCRIPTION
**Pull request by danielemarostica**
## Pull Request Type

- [ ] Bugfix
- [x] Documentation content changes

## Description

Added documentation for the new User Preferences section in Console

## PR Checklist

- [x] All tests of Lint, cspell and check-content pass. ([How to launch them?](../CONTRIBUTING.md#content-checks))
- [x] No sensible content has been committed

### CODE REVIEW

**Review by fredmaggiowski — APPROVED** [review]
LGTM

Must be featured with v10.6.0 so we have to wait before merging it

File: docs/development_suite/user-settings/user-preferences.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+id: user-preferences
+title: User Preferences
+sidebar_label: User Preferences
+---
+
+### Manage your Console Preferences
+Hovering over the user profile picture, from the dropdown menu, click on the `User Preferences` item to view and manage some behaviors of the console for your account.
```
Comment (albertotessarotto) [inline docs/development_suite/user-settings/user-preferences.md:8]:
```suggestion
Hovering over the user profile picture, from the dropdown menu, click on the `User Preferences` item to view and manage some behaviors of the Console for your account. 
```

File: docs/development_suite/user-settings/user-preferences.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+id: user-preferences
+title: User Preferences
+sidebar_label: User Preferences
+---
+
+### Manage your Console Preferences
+Hovering over the user profile picture, from the dropdown menu, click on the `User Preferences` item to view and manage some behaviors of the Console for your account.
```
Comment (marcofilippi) [inline docs/development_suite/user-settings/user-preferences.md:8]:
```suggestion
After hovering over the user profile picture, from the dropdown menu, click on the `User Preferences` item to view and manage some preferences in Console for your account. 
```

File: docs/development_suite/user-settings/user-preferences.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+sidebar_label: User Preferences
+---
+
+### Manage your Console Preferences
+Hovering over the user profile picture, from the dropdown menu, click on the `User Preferences` item to view and manage some behaviors of the Console for your account. 
+
+![user preferences dropdown](./img/user-preferences-dropdown.png)
+
+The preferences that you can manage are:
+- **Company invitation email**: this setting manages whether or not you receive an email notification when you are added on a new Company.
```
Comment (marcofilippi) [inline docs/development_suite/user-settings/user-preferences.md:13]:
```suggestion
- **Company invitation email**: through this setting you can decide whether or not to receive an invitation email when you are added on a Company.
```

File: docs/development_suite/user-settings/img/user-preferences-dropdown.png
Comment (marcofilippi) [inline docs/development_suite/user-settings/img/user-preferences-dropdown.png:1]: da aggiornare screen: ora abbiamo nuove voci di menù

File: docs/development_suite/user-settings/img/user-preferences-page.png
Comment (marcofilippi) [inline docs/development_suite/user-settings/img/user-preferences-page.png:1]: da aggiornare screen: mostriamo il toggle abilitato

**Pull request merged**